packages/forma-36-codemod/transforms/v4-badge.js

```diff
@@ -0,0 +1,56 @@
+const {
+  getComponentLocalName,
+  renameProperties,
+  changeProperties,
+  changeImportWithComponentName,
+  changeComponentName,
+} = require('../utils');
+const { getFormaImport, shouldSkipUpdateImport } = require('../utils/config');
+
+module.exports = function (file, api) {
+  const j = api.jscodeshift;
+
+  let source = file.source;
+
+  const componentName = getComponentLocalName(j, source, {
+    componentName: 'Tag',
+    importName: getFormaImport(),
+  });
+
+  if (!componentName) {
+    return source;
+  }
+
+  source = changeProperties(j, source, {
+    componentName,
+    fn(attributes) {
+      let modifiedAttributes = attributes;
+
+      modifiedAttributes = renameProperties(modifiedAttributes, {
+        renameMap: {
+          tagType: 'variant',
```

Comment:
do we have the same variant values as they were in tagType? no renamed values?

Reply:
yeah, they are the same. 
But now when I think about it, we should actually remove `muted` since we deprecate it in v3. I will confirm that with the designers. but this change would need to be done in the API of the v4 Badge.
```
export type BadgeVariant =
  | 'negative'
  | 'positive'
  | 'primary'
  | 'secondary'
  | 'warning'
  | 'muted'
  | 'primary-filled'
  | 'featured';
```